bundles/org.openhab.binding.enocean/src/main/resources/ESH-INF/thing/LightSensor.xml

```diff
@@ -18,13 +18,15 @@
 				<description>EnOceanId of device this thing belongs to</description>
 				<required>true</required>
 			</parameter>
-			<parameter name="receivingEEPId" type="text">
+			<parameter name="receivingEEPId" type="text" multiple="true">
```

Comment:
Buts this also allows to select `A5_06_01` and `A5_06_01_ELTAKO` which is probably not what you intended. What happens if the user does that? (also applies below)

Reply:
As the choosen EEPs define the channels of a thing I added the D0_06_00 EEP to the receivingEEP too. Later during validation of the thing configuration I am checking if the user has choosen max one EEP of each EEP family (RPS, 1BS, 4BS, VLD, SIG).
So it is true that I do not want the user to select ```A5_06_01``` and ```A5_06_01_ELTAKO```, but adding a new config param for each multi EEP receiving case would have made the implementation a little bit more complicated.

Reply:
Can you point me to the code that validates this`?

Reply:
Yes for sure, validation is done in the EnOceanBaseSensorHandler, function [validateConfig()](https://github.com/openhab/openhab-addons/blob/469a6ae822ea7987d3bb2c438a99c657510a8678/bundles/org.openhab.binding.enocean/src/main/java/org/openhab/binding/enocean/internal/handler/EnOceanBaseSensorHandler.java#L82) Line 95

Reply:
I prepared a PR against your fork. WDYT?

Reply:
@J-N-K  Thanks for the proposal. I really like your approach. I will accept your PR, rework the thing xmls and update the PR. Thanks a lot. By the way, I am always learning something new from your code, was not aware of the shorthand function pointer/lambda in line 158 👍 .

Reply:
@J-N-K thanks again for your proposal. I updated my PR accordingly.